docs/source/references.bib

```diff
@@ -140,19 +132,24 @@ @Book{Helmert1880
   Doi                      = {10.5281/zenodo.32050}
 }
 
-@Article{Hensley2002,
-  Title                    = {Improved Processing of AIRSAR Data Based on the GeoSAR Processor},
-  Author                   = {Scott Hensley and Elaine Chapin and Adam Freedman and Thierry Michel},
-  Journal                  = {Airsar earth science and applications workshop},
+@InProceedings{Hensley2002,
+  Title                    = {Improved Processing of {AIRSAR} Data Based on the {GeoSAR} Processor},
+  Author                   = {S. Hensley and E. Chapin and A. Freedman and T. Michel},
```

Comment:
I'm not sure there's any benefit to reducing the information here. Whether or not the full name appears is up to the bibliography style in the end, but I don't think there's any reason there should _not_ be full first names.

Reply:
In regards to reducing first names to initials, I see your point. I mainly did this for consistency with the initial bibliography that just used initials. If we kept some entries with first names, it may look a bit more chaotic.

Reply:
Consistency is better than inconsistency. If reducing a few first names to initials is the simpest way to achieve consistency that is what should be done.

Reply:
I **strongly** disagree here: The bib-file is a database of available information, and should primarily register information **as published**. 

The bibliography style a different thing: Removing information from the database is patching the problem at the wrong end. Better, for some time have a mix of initials and full names - if we want initials only, the bib-style should enforce that.

Reply:
Yeah, I was split on this one. The idea from ~8 months ago was to make the references section Harvard-like, which uses initials. Usually it should be up to the bib style to decide how to format this, but the current structure doesn't seem to support this. I believe this support could eventually happen, by extending parts of `docs/source/bibstyle.py`.

So I suppose the options are to either keep with hard-coded initials (for now), or mix the citation style (initials/full names). And if I do eventually tidy the bib style to format initials, then the .bib file can be revised to reflect names as published.

Reply:
Ah, I seem to have missed that the [API does have a `abbreviate_names` option with default of False](https://pythonhosted.org/pybtex/api/styles.html). So this won't be too difficult to do after all. However, could this wait until after the release (i.e. next week)? Basically, I'd fix the citation style and go thru all the references to check the consistency with publications (where feasible).

Reply:
> However, could this wait until after the release (i.e. next week)?

Of course. Also note that the documentation isn't part of the release tarball so the deadline for inclusion of contributions in 6.0 is more fluid than for code contributions. We can merge this now and you can fix the names later, or we can wait a bit until you are ready. Whatever you prefer.

Reply:
@kbevers yes, this is ready to merge. I'll revise names as published for the .bib file later.